scripts/generate/templates/Checkpoints.js

```diff
@@ -146,6 +166,12 @@ function _insert(
     }
 }
 
+/**
+ * @dev Return the index of the last checkpoint for which \`key\` >= checkpoint.key, i.e. the most recent checkpoint which key is lower or equal than the search key.
```

Comment:
This is not right! It is not the most recent with lower or equal key. It is the next one after that one, so the _least_ recent with _greater_ key.

```suggestion
 * @dev Return the index of the oldest checkpoint whose key is greater than the search key, or `high` if there is none.
```